src/els_group_leader_server.erl

```diff
@@ -0,0 +1,126 @@
+%%==============================================================================
```

Comment:
It's nice to have a remote node, but things seem to start getting really complicated now. 😄

Reply:
It is indeed a massive complication to the architecture, especially now that other portions of the code are not using it (e.g. code reload, compilation). It feels very much like when we introduced Mnesia and persistent storage. I suspect it will take a while until we stabilize again. But please report any issue you find with the new setup.